package/MDAnalysis/core/universe.py

```diff
@@ -1459,13 +1490,109 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
                 "hydrogens with `addHs=True`")
 
             numConfs = rdkit_kwargs.pop("numConfs", numConfs)
-            if not (type(numConfs) is int and numConfs > 0):
+            if not (isinstance(numConfs, int) and numConfs > 0):
                 raise SyntaxError("numConfs must be a non-zero positive "
-                "integer instead of {0}".format(numConfs))
+                                  "integer instead of {0}".format(numConfs))
             AllChem.EmbedMultipleConfs(mol, numConfs, **rdkit_kwargs)
 
         return cls(mol, **kwargs)
 
+    def guess_TopologyAttrs(
+            self, context=None, to_guess=(), force_guess=(), **kwargs):
+        """
+        Guess and add attributes through a specific context-aware guesser.
+
+        Parameters
+        ----------
+        context: str or :mod:`Guesser<MDAanalysis.guesser>` class
+            For calling a matching guesser class for this specific context
+        to_guess: list[str] (optional, default ``['types', 'masses']``)
```

Comment:
Two things here:

1. Optionals should really be `None` rather than `()`, it's a nit-pick but only in the sense that Optional[List] means Union[None, List] in Python
2. From the signature, the default is `()` not `['types', 'masses']`.